python/ray/autoscaler/gcp/example-full.yaml

```diff
@@ -85,7 +85,7 @@ worker_nodes:
         initializeParams:
           diskSizeGb: 50
           # See https://cloud.google.com/compute/docs/images for more images
-          sourceImage: projects/ml-images/global/images/family/tf-1-13-gpu
+          sourceImage: projects/deeplearning-platform-release/global/images/family/tf-1-13-cpu
```

Comment:
Related to #4524. Seems like `deeplearning-platform-release` is more standard than `ml-images`. I also changed this from gpu to cpu to be consistent with the aws example. cc @toanngosy.

Reply:
Thanks @hartikainen!